test/scrabble_test.rb

```diff
@@ -5,9 +5,41 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
+
+  attr_reader :scrabble
+
+  def setup
+    @scrabble = Scrabble.new
+  end
+
+  def test_class_exists
+    assert_instance_of Scrabble, scrabble
+  end
+
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+
+  def test_it_can_score_a_word
+    assert_equal 4, scrabble.score("anna")
+    assert_equal 8, scrabble.score("word")
```

Comment:
I like that you tested multiple words; could have improved robustness (is that a word? :) ) by testing words of different length.